.github/workflows/build.yml

```diff
@@ -135,3 +136,42 @@ jobs:
         with:
           name: v3-linux
           path: linux.tar.gz
+  v3-macOS:
+      runs-on: macos-latest
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+            submodules: recursive
+        - name: Download libogg, libvorbis, and libtheora for easier includes
+          run: |
+            curl "https://ftp.osuosl.org/pub/xiph/releases/ogg/libogg-1.3.5.zip" --output "libogg.zip"
+            unzip libogg.zip -d ./dependencies/mac/
+            mv ./dependencies/mac/libogg-1.3.5 ./dependencies/mac/libogg
+            curl "https://ftp.osuosl.org/pub/xiph/releases/vorbis/libvorbis-1.3.7.zip" --output "libvorbis.zip"
+            unzip libvorbis.zip -d ./dependencies/mac/
+            mv ./dependencies/mac/libvorbis-1.3.7 ./dependencies/mac/libvorbis
+            curl "https://ftp.osuosl.org/pub/xiph/releases/theora/libtheora-1.1.1.zip" --output "libtheora.zip"
+            unzip libtheora.zip -d ./dependencies/mac/
+            mv ./dependencies/mac/libtheora-1.1.1 ./dependencies/mac/libtheora
+        - name: Download SDL2
+          run: |
+            curl "https://libsdl.org/release/SDL2-2.30.7.dmg" --output "SDL2.dmg"
+            hdiutil attach SDL2.dmg
+            cp -r /Volumes/SDL2/SDL2.framework ./dependencies/mac
+        - name: Generate Vorbis headers
+          working-directory: ./dependencies/mac/libogg/
+          run: |
+            ./configure
+        - name: Build RSDKv3
+          run: |
+            xcodebuild -project RSDKv3.xcodeproj -scheme RSDKv3 DSTROOT="./build" -configuration Release DEPLOYMENT_LOCATION=YES ${{env.GENERAL_MACOS_FLAGS}}
+        - name: Zip the app ourselves (Important step to retain symlinks)
+          run: |
+            cd ./build/Applications/
+            zip -r RSDKv3.zip ./RSDKv3.app/
```

Comment:
Since `xattr -cr RSDKv3.app` is required for anyone to run the decomp on their machine, you could add a simple README in the zip file:
```sh
echo 'Install the .app file and run "xattr -cr /Applications/RSDKv3.app" in the terminal' > README.txt
```
And perhaps add this instruction somewhere in the build guide?

Reply:
That I can do

Reply:
Do you think this is good?
```
cd ./build/Applications/
echo 'From this folder, run `xattr -cr RSDKv3.app` in Terminal to allow MacOS to open it.' > README.txt
zip -r RSDKv3.zip .
```
Because I included the readme in the zip, it extracts the folder RSDKv3 and it has the readme and app inside.
<img width="139" alt="Screenshot 2024-09-09 at 12 12 53 PM" src="https://github.com/user-attachments/assets/d9d0caaa-8750-4cc4-b8b5-1866341f7ac8">